crates/subspace-service/Cargo.toml

```diff
@@ -52,6 +53,7 @@ sc-telemetry = { git = "https://github.com/subspace/polkadot-sdk", rev = "9b8cdb
 sc-tracing = { git = "https://github.com/subspace/polkadot-sdk", rev = "9b8cdb87de8f1c0e6b48c468b6196d1d99eeabee" }
 sc-transaction-pool = { git = "https://github.com/subspace/polkadot-sdk", rev = "9b8cdb87de8f1c0e6b48c468b6196d1d99eeabee" }
 sc-transaction-pool-api = { git = "https://github.com/subspace/polkadot-sdk", rev = "9b8cdb87de8f1c0e6b48c468b6196d1d99eeabee" }
+sc-utils = { git = "https://github.com/subspace/polkadot-sdk", rev = "808269708cf5375526755797e8f9a9986016727d" }
```

Comment:
This dependency doesn't seem to be used, it is also not the same version as the rest of dependencies, which caused significant (and undesired) changes of the lock file

Reply:
It doesn't cause compile-time errors but it would cause weird errors in the future. Thanks for catching it!